src/test/java/com/google/longrunning/OperationsClientTest.java

```diff
@@ -228,7 +228,7 @@ public void deleteOperationExceptionTest() throws Exception {
       client.deleteOperation(name);
       Assert.fail("No exception raised");
     } catch (ApiException e) {
-      Assert.assertEquals(Status.INTERNAL.getCode(), e.getStatusCode());
+      Assert.assertEquals(Status.INVALID_ARGUMENT.getCode(), e.getStatusCode());
```

Comment:
I was referring specifically to the previous generation changes - including the status code change for tests + the versioning stuff, but not your paging changes. Your generated paging changes of course need to be included in the same PR.